Spanner/src/Operation.php

```diff
@@ -339,13 +363,26 @@ public function executeUpdateBatch(
             ] + $this->mapper->formatParamsForExecuteSql($parameters, $types);
         }
 
+        if (!is_null($transaction->id())) {
+            $options += ['transactionId' => $transaction->id()];
+        }
+
```

Comment:
same here, this is giving `$options['transactionId']` a higher priority over the `$transaction->id()`. Is this what we want? And if so, is the `!is_null` check really necessary?

Reply:
Please see the above answer. Also, will change this to avoid confusion:
```
if (!isset($options['transaction']['begin'])) {
    $options = ['transactionId' => $transaction->id()] + $options;
}
```

Reply:
Done!